package.json

```diff
@@ -20,10 +20,11 @@
     "clean:packager": "watchman watch-del-all && rm -rf $TMPDIR/react-* && rm -rf $TMPDIR/metro-* && rm -rf $TMPDIR/haste-map-*",
     "clean:node": "rm -rf node_modules",
     "nuke": "./scripts/nuke.sh",
-    "detox:android": "detox build -c android.emu.debug && detox test -R 1 -c android.emu.debug --loglevel verbose",
-    "detox:android:release": "detox build -c android.emu.release && detox test -R 1 -c android.emu.release",
-    "detox:ios": "detox build -c ios.sim.debug | xcpretty --color && detox test -R 1 -c ios.sim.debug --bail",
-    "detox:ios:release": "detox build -c ios.sim.release && detox test -R 1 -c ios.sim.release --bail",
+    "detox:android": "detox build -c android.emu.debug && detox test -c android.emu.debug --loglevel verbose",
+    "detox:android:release": "detox build -c android.emu.release && detox test -c android.emu.release",
+    "detox:ios:tests": "detox test -c ios.sim.debug --maxWorkers 2 -- --bail 1",
+    "detox:ios": "detox build -c ios.sim.debug | xcpretty --color && yarn detox:ios:tests",
+    "detox:ios:release": "detox build -c ios.sim.release && detox test -c ios.sim.release --maxWorkers 2 -- --bail 1",
```

Comment:
what happens when maxWorkers > 2?

Reply:
more parallelization, but diminishing returns obvs. we have 6 tests RIGHT NOW with more to come.

1 worker = 16 mins
2 workers = 8 mins
3 workers = 5.33 mins

right now 4+ workers is awkward bc the tests become uneven but it's still faster. 

once we add more workers and split up the tests more i think the flakiness could become a concern bc of how split resources become. i'll see how far i can push it tho! i think at least 3 is probably good.

Reply:
yeah i tried it out it seemed like more than 2 workers currently introduces some flakiness